lingua_franca/lang/parse_de.py

```diff
@@ -1156,6 +1156,14 @@ class GermanNormalizer(Normalizer):
     with open(resolve_resource_file("text/de-de/normalize.json")) as f:
         _default_config = json.load(f)
 
+    def remove_symbols(self, utterance):
```

Comment:
this method should go into the base normalizer and get a flag for all langs in the .json

the more performant way would be
```
text = "15-minute-timer"
translating = str.maketrans('', '', string.punctuation)
new_string = text.translate(translating)
```

Reply:
`str.maketrans(''.join(symbols), len(symbols)*" ")`  resp.
 `str.maketrans(symbols, len(symbols)*" ")`  # second param has to be of the same length

Will add this tomorrow